client/src/components/FeedbackPage/FeedbackTestimonialsSection/FeedbackTestimonialsSection.tsx

```diff
@@ -101,8 +52,8 @@ const FeedbackTestimonialsSection: FunctionComponent = () => {
           <h2>Testimonials</h2>
           {data.length > 0 ? (
             <RequestDataWrapper data={data} loading={loading} error={error}>
-              {testimonialsArray?.map(({ id, content }, index) => (
-                <Accordion key={id} title={content.title} index={index}>
+              {testimonialsDataArray?.map(({ id, content }, index) => (
+                <Accordion key={id} title={testimonialsConfigArray[index].title} index={index}>
```

Comment:
using `testimonialsConfigArray` you need always keep in mind that items ordering is very important. would be better to use `testimonialsConfig[id].title` instead

Reply:
Ups, my bad. You're definitely right!:)